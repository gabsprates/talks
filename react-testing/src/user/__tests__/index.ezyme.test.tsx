import React from "react";
import { User } from "../index";
import { shallow } from "enzyme";

describe("Component: User", () => {
  it("should render without errors", () => {
    const wrapper = shallow(<User name="prates" github="gabsprates" />);

    expect(wrapper.html()).toMatchSnapshot();
    expect(wrapper.find("strong").text()).toMatch("prates");
  });
});
