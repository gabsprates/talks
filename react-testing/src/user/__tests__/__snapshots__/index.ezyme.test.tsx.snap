// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: User should render without errors 1`] = `"<div class=\\"user\\"><strong class=\\"name\\">prates</strong><div class=\\"github\\">gabsprates</div></div>"`;
